packages/ai/core/generate-image/generate-image.test.ts

```diff
@@ -61,7 +61,13 @@ describe('generateImage', () => {
       size: '1024x1024',
       aspectRatio: '16:9',
       seed: 12345,
-      providerOptions: { openai: { style: 'vivid' } },
+      providerOptions: {
+        'mock-provider': {
+          style: 'vivid',
+          // maxImagesPerCall is not passed to doGenerate
+          maxImagesPerCall: 3,
+        },
+      },
```

Comment:
I assume the intend was to pass `'mock-provider'` as `providerOptions` key to match the mock provider? Or is there a valid use case to pass `providerOptions` that don't match the passed model?